src/views/Profile/OperationView.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Container } from '@material-ui/core';
+import { MaterialsView } from './MaterialsView';
+import { useStyles } from './styles/OperationView.styles';
+import { IProfileMenuOption } from '../../old/lib/types';
+
+interface IOperationViewProps {
+  selectedOption: IProfileMenuOption | Record<string, never>;
+}
+
+export const OperationView: React.FC<IOperationViewProps> = (props) => {
+  const { selectedOption } = props;
+  const { value } = selectedOption;
+  const classes = useStyles();
+
+  const renderOperationView = () => {
```

Comment:
It  would be cleaner to use dispatching here

```jsx
  const operationViews = {
    info: <div>Info</div>,
    operationView: <MaterialsView />,
    passwordChange: <div>Password Change</div>,
    mail: <div>Mail</div>,
  };
  return (
    <Container className={classes.operationView}>
      {operationViews[value] ?? <div>Info</div>}
    </Container>
  );

```

Reply:
Done (yes, it's much more better)